packages/common/src/styles/RTLProvider.tsx

```diff
@@ -4,12 +4,9 @@ import { useRtl } from '@common/intl';
 export const RTLProvider: FC = props => {
   const isRtl = useRtl();
 
-  React.useLayoutEffect(() => {
-    document.body.setAttribute('dir', isRtl ? 'rtl' : 'ltr');
-  }, [isRtl]);
-
   return (
     <div
+      dir={isRtl ? 'rtl' : 'ltr'}
```

Comment:
same effect as adding to the body (probably nothing), just feels better to use declaratively rather than an effect.